src/Paprika/Store/PageManagers/MemoryMappedPageManager.cs

```diff
@@ -24,6 +27,18 @@ public sealed class MemoryMappedPageManager : PointerPageManager
     private readonly MemoryMappedViewAccessor _whole;
     private readonly unsafe byte* _ptr;
 
+    // Prefetcher
```

Comment:
Ah, it could be made more specific as this is about the `Sequential` flag as we don't do `Sequential` because clearly it's a random access all over the place. Will amend.